netbout-web/src/main/java/com/netbout/dynamo/DyAttachment.java

```diff
@@ -163,6 +164,20 @@ public boolean unseen() throws IOException {
     }
 
     @Override
+    public Date date() throws IOException {
+        return new Date(
+            Long.parseLong(
+                this.item.get(DyFriends.ATTR_UPDATED).getN()
```

Comment:
@CarlosGines have you investigated whether it is possible to keep date in some proper date format here?